lms/static/scripts/frontend_apps/components/OAuth2RedirectErrorApp.js

```diff
@@ -23,20 +23,32 @@ export default function OAuth2RedirectErrorApp({ location = window.location }) {
     OAuth2RedirectError: {
       authUrl = /** @type {string|null} */ (null),
       invalidScope = false,
+      errorCode = '',
       errorDetails = '',
       canvasScopes = /** @type {string[]} */ ([]),
     },
   } = useContext(Config);
 
-  const title = invalidScope
-    ? 'Developer key scopes missing'
-    : 'Authorization failed';
+  const title = () => {
+    if (invalidScope) {
+      return 'Developer key scopes missing';
+    }
 
-  const message = invalidScope
-    ? null
-    : 'Something went wrong when authorizing Hypothesis';
+    if (errorCode === 'blackboard_missing_integration') {
+      return 'Missing blackboard REST API Integration';
+    }
 
-  const error = { details: errorDetails };
+    return 'Authorization failed';
+  };
+
+  const error = { code: errorCode, details: errorDetails };
+
+  const message = () => {
+    if (invalidScope || errorCode !== null) {
+      return ' ';
```

Comment:
This is odd and leads to a random "." to show up in the dialog. Using `null` as before leads to the message `An unknown error occurred.` to show up which is not enterily correct (it's a known error)

This seem to come from here:https://github.com/hypothesis/lms/blob/4c5bad17e096a7e9ea949cbab667efc1856ed63a/lms/static/scripts/frontend_apps/components/ErrorDisplay.js#L109

and I haven't found a way to just not show anything. I could below use a conditional and only use `<ErrorDisplay message={message()} error={error} />`in the `else` part but `ErrorDisplay` is responsible of showing the "contact support" part as well. Maybe a new bool attribute for `ErrorDisplay` to disable this message?